### PR TITLE
Also use ubuntu-latest for the FreeBSD build

### PR DESCRIPTION
**Pull request by melroy89**
Let's also use the `ubuntu-latest` run tag for the FreeBSD which is still Ubuntu 22.04, but most likely soon Ubuntu 24.04: https://github.com/actions/runner-images

### CODE REVIEW

**Comment by Neustradamus** [comment]
@RICCIARDI-Adrien: What do you think?

Last year:
- https://github.com/radvd-project/radvd/pull/217

**Comment by stappersg** [comment]
On Thu, Dec 05, 2024 at 08:57:31PM -0800, Neustradamus wrote:
> @RICCIARDI-Adrien: What do you think?
> 
> Last year:
> - https://github.com/radvd-project/radvd/pull/217
> 

@Neustradamus please remember
that you have approval privilege for this project.


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by RICCIARDI-Adrien** [comment]
Hi !
The last release of the FreeBSD action (https://github.com/vmactions/freebsd-vm/tree/v1.1.5) is indeed running on Ubuntu 24.04.
I would suggest to also update `uses: vmactions/freebsd-vm@v1.0.2` to `uses: vmactions/freebsd-vm@v1` so we always use the last action version, hoping it won't break.

**Comment by melroy89** [comment]
> Hi ! The last release of the FreeBSD action (https://github.com/vmactions/freebsd-vm/tree/v1.1.5) is indeed running on Ubuntu 24.04. I would suggest to also update `uses: vmactions/freebsd-vm@v1.0.2` to `uses: vmactions/freebsd-vm@v1` so we always use the last action version, hoping it won't break.

Agreed. Updated as well!

Assuming they do not introduce breaking changes in the same major release (which they shouldn't).

**Comment by RICCIARDI-Adrien** [comment]
Looks good to me !

**Review by RICCIARDI-Adrien — APPROVED** [review]

**Comment by melroy89** [comment]
It seems you & I have no approval right (meaning you will need at least write access to this org or to this repo).

**Review by stappersg — APPROVED** [review]
OK

**Comment by stappersg** [comment]
This _approve merge request_ is a **lead by example** reference to #237

**Pull request merged**
